app/scripts/directives/bindService.js

```diff
@@ -189,9 +192,38 @@
       });
     };
 
+    var loadMobileServiceClass = function(svcToBind) {
```

Comment:
should this be named loadMobileServiceClass? Is svcToBind actually the service instance.  Is there no loadServiceClassfromServiceInstance or something like that in the catalog service? Seems like this is generally useful to do and not mobile specific

```
sc = catalogService.getClusterServiceClassFromServiceInstance(si)
if (isMobileService(sc)){
...

}

```

Reply:
looking through I see https://github.com/JameelB/origin-web-console/blob/b8798722dc96a7f12853baf41ad81bebf524454c/app/scripts/directives/bindService.js#L179 which seems it would do what we need?

Reply:
The only thing that I could find that does this is the [getServiceClassForInstance](https://github.com/openshift/origin-web-common/blob/master/src/services/bindService.js#L17) from the BindService, in a similar way they do it [here](https://github.com/openshift/origin-web-console/blob/master/app/scripts/directives/bindService.js#L244). For this to work, we would need to get a list of the serviceClasses first. We can get a list of all the serviceClasses here and rename it to loadServiceClasses and let [this](https://github.com/openshift/origin-web-console/blob/master/app/scripts/directives/bindService.js#L244) handle getting the serviceClass associated with the serviceInstance.

As far as I understood, that `svcToBind` should always be a serviceInstance because they check the type [here](https://github.com/openshift/origin-web-console/blob/master/app/scripts/directives/bindService.js#L201)